### PR TITLE
[2.1] Robot integrations: Fixes for UR articles

### PR DESCRIPTION
**Pull request by adolfo-rt**
Apart from content fixes, this PR addresses all doc generation warnings raised by the UR articles.

### CODE REVIEW

File: docs/robot-integrations/universal-robots/urcap/getting_started_with_the_pickit_urcap.rst [hunk 1]
```diff
@@ -93,7 +93,7 @@ If you plan to run robot programs that don't use Pickit, you should disable (not
 Example programs
 ----------------
 
-The Pickit URCap plugin installation makes available a few example programs under ``/programs/pickit_samples`` folder of the robot. The can be also downloaded independently of the URCap \ `here <https://drive.google.com/open?id=1Gf63Y35NaVxbP4mwc5YUC5SU8u8RYvyO>`__. These programs are a great way to get familiar with the Pickit URCap plugin, and can serve as a template to build your own applications. The following articles provide detailed descriptions of example programs:
+The Pickit URCap plugin installation makes available a few example programs under ``/programs/pickit_samples`` folder of the robot. The can be also downloaded independently of the URCap \ `here <https://drive.google.com/uc?export=download&id=1Gf63Y35NaVxbP4mwc5YUC5SU8u8RYvyO>`__. These programs are a great way to get familiar with the Pickit URCap plugin, and can serve as a template to build your own applications. The following articles provide detailed descriptions of example programs:
```
Comment (adolfo-rt) [inline docs/robot-integrations/universal-robots/urcap/getting_started_with_the_pickit_urcap.rst]: Use direct download.

File: docs/robot-integrations/universal-robots/urcap/index.rst [hunk 2]
```diff
@@ -7,4 +7,8 @@ URCap
   :glob:
   :maxdepth: 1
 
-  *
+  getting_started_with_the_pickit_urcap.rst
```
Comment (adolfo-rt) [inline docs/robot-integrations/universal-robots/urcap/index.rst]: Reorder items how the user should read them.

File: docs/robot-integrations/universal-robots/urcap/robot-camera-calibration-with-the-urcap-plugin.rst [hunk 3]
```diff
@@ -19,6 +17,7 @@ The program starts by opening a pop-up message, informing that multi-poses calib
 #. Moves the robot to a waypoint
 
    .. image:: /assets/images/robot-integrations/ur/urcap-calibration-2.png
+      :scale: 50 %
```
Comment (adolfo-rt) [inline docs/robot-integrations/universal-robots/urcap/robot-camera-calibration-with-the-urcap-plugin.rst]: For some reason, some images were rendered out of scale, twice as large.

File: docs/robot-integrations/universal-robots/urcap/universal_robots_urcap_example_picking_program.rst [hunk 4]
```diff
@@ -104,7 +104,7 @@ Running the program
 -------------------
 
 Program execution
-^^^^^^^^^^^^^^^^^
+~~~~~~~~~~~~~~~~~
```
Comment (adolfo-rt) [inline docs/robot-integrations/universal-robots/urcap/universal_robots_urcap_example_picking_program.rst]: The title hierarchy was not respected here, and these titles were not being rendered.

**Comment by anaritaflp** [comment]
@adolfo-rt Thank you for fixing this! I'm posting some of my review comments here, because they are not on your changes, but on text that was already there.

### `getting_started_with_the_pickit_urcap.rst` 
- L30: Download link leads to Documentation page instead of downloads page.
- L39: The link in "Download" leads to a 404 page not found.
- L50: Instructions overlap with previous step. Ex., should be just "Press + to install a new URCap."

### `universal_robots_urcap_example_picking_program.rst `
- In lists, it's sometimes missing a full stop at the end of each list item.
    - In section `Program breakdown`, where it lists the possible cases after detection.
    - In section `Object detected and reachable`, when describing each step of the sequence.
- In section `Monitoring camera disconnections`: "When this is the case, it typically indicates **a a** hardware disconnection issue..."

### `robot-camera-calibration-with-the-urcap-plugin.rst`
- Section `Multi poses calibration`: Missing full stop at the end of some sentences in the listed items.
- Section `Multi poses calibration`: Item 2 does not describe an extra action, and its content should be added to item 1.

### `the_pickit_urcap_interface.rst`
- In box `Check if robot mode enabled`: "Except for Find calibration plate, all other Pickit plugin commands require robot mode to be ~~Robot mode is~~ enabled."
- In box `Get next object`: "The next Pickit command after  Find object(s) should always be ~~Get next object,~~ Get result, which waits until a response for the request is ready."

### `urscript/index.rst`
- The link in section `Download the right files` leads to a 400 Bad request page.

**Comment by adolfo-rt** [comment]
Comments addressed, download links should already be fixed in 2.1 by other recent PRs.

**Pull request merged**